### DISCUSSION TITLE
Feature Request: Aurora Serverless

### DISCUSSION THREAD

**Discussion by dave-graham** [aws/serverless-application-model#3528] [Ideas]
Hey,

Given the availability of aurora serverless and aurora serverless data api (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html) making it simpler to manage permissions, are there any plans to pull aurora serverless into the SAM setup?

Dynamo is great for a number of things, but every now and then it would be useful to have a database we can run joins on.

Would also be great to have some aurora functionality in the SAM CLI.

If I have to set this up via another route, thats fine, just it would be nice to have all the infrastructure we can use for serverless in one place, as well as incorporate a number of bestg practice pieces that SAM provides.

**Comment by keetonian** [comment]
You can use the native cloudformation resources for Aurora in a SAM template. Are there any abstractions that you have in mind that you would like SAM to make for you when setting up an Aurora database?
